operators/src/lib.rs

```diff
@@ -1,4 +1,5 @@
 #![feature(async_closure)]
+#![feature(available_concurrency)]
```

Comment:
Why do we need another unstable feature?

Reply:
it's supposed to be the best way to determine the ideal number of threads